README.md

```diff
@@ -33,6 +33,7 @@ apiVersion: jaegertracing.io/v1
 kind: Jaeger
 metadata:
   name: simplest
+  namespace: observability
```

Comment:
We should be consistent with other manifests and do not harcode the namespace.  `apply -n observability` should do the same.